### DISCUSSION TITLE
Allow hiding "Aborted" on CTRL+C

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [fastapi/typer#852] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorials in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
-
```

### Description

When pressing CTRL+C, the message 'Aborted' is shown. This can be confusing. Therefore, it should be possible to not print that message. 

This behaviour comes from Click, but it's not trivial to work around it: https://github.com/pallets/click/issues/2112. Typer seems like the ideal place to add a 'simple' toggle, abstracting the implementation from its users. 

### Operating System

Linux

### Typer Version

0.12.3

### Python Version

3.11.6

**Comment by svlandeg** [comment]
Can you explain a bit more when this happens exactly? Is using `typer.Exit()` as explained [here](https://typer.tiangolo.com/tutorial/terminating/) an option?

**Reply by WilliamDEdwards** [reply]
> Can you explain a bit more when this happens exactly?

This happens when pressing CTRL+C. 

> Is using `typer.Exit()` as explained [here](https://typer.tiangolo.com/tutorial/terminating/) an option?

No, I can't intercept Click's hook AFAIK.

**Reply by YuriiMotov** [reply]
I also can't reproduce it (in current version - 0.17.4).
@WilliamDEdwards, could you please provide a code example that we can use to reproduce displaying the `Aborted` message on CTRL+C?
